conf/set_config.go

```diff
@@ -15,60 +15,94 @@ import (
 	. "github.com/logrusorgru/aurora"
 )
 
-// SetConfigValueByArgs updates the config file by merging a single new value with the
-// current in memory configuration.
+// SetConfigValueByArgs updates the config file by merging a **single** new value with the
+// current in memory configuration. Once merged, it writes the updated config to disk
 // - args are a slice of new values such as `[]string{"nginx", "workerconnections", "30"}`
-// - file is either 'project' or 'global' and will determine which file is updated
-func SetConfigValueByArgs(args []string, configFile string) {
-	if configFile != "project" && configFile != "global" {
-		fmt.Println(Sprintf("The config file %s is unknown", Bold(configFile)))
+// - configType is either 'project' or 'global' and will determine which file is updated
+func SetConfigValueByArgs(args []string, configType string) {
+	if configType != "project" && configType != "global" {
+		fmt.Println(Sprintf("The config file %s is unknown", Bold(configType)))
 		os.Exit(1)
 	}
 
 	validateArgs(args)
 
-	yt := argsToYaml(args)
-	cp := getConfigPath(configFile)
+	newYaml := argsToYaml(args) // carries yaml-formatted string of singular args slice
+	configPath := getConfigPath(configType)
```

Comment:
Sometimes it's good to have meaningful names, Golang best practice guide be damned